.changes/v0.12.0.md

```diff
@@ -0,0 +1,77 @@
+## v0.12.0 - 2024-07-12
+
+This release is significant. All details (including videos & code examples)
+are captured [in this blog post](https://dagger.io/blog/dagger-0-12).
+
+### 🔥 Breaking Changes
+
+- sdks: Various breaking changes to SDKs \
+  See SDK-specific release notes for more information
+- api: Align `Container.withNewFile` signature with `Directory.withNewFile` by @helderco in https://github.com/dagger/dagger/pull/7293 \
+  Callers of `Container.withNewFile` will need to be updated depending on SDK.
+- api: Skip entrypoint by default in `withExec` by @helderco in https://github.com/dagger/dagger/pull/7136 \
+  Callers relying on a `Container`s default entrypoint will need to be updated
+  to opt-in with `UseEntrypoint`.
+- api: `Container.terminal` now returns a `Container` by @aluzzardi in https://github.com/dagger/dagger/pull/7586 \
+  Callers of `terminal` will need be updated to handle the updated type.
+- api: Return absolute path on export instead of boolean by @helderco in https://github.com/dagger/dagger/pull/7500 \
+  Callers expecting a boolean return will need to be updated to instead
+  handle a string path.
+- api: Don't fallback to the default command on `Container.stdout` and `Container.stderr` by @helderco in https://github.com/dagger/dagger/pull/7857 \
+  Callers of `stdout` and `stderr` without a previous `withExec` will need to
+  insert an explicit empty `withExec`.
```

Comment:
Move this next to the entrypoing breaking change. If you also relied on the entrypoint in this case, you'll need both: empty args and `useEntrypoint`:
```go
build.WithExec([]string{}, dagger.ContainerWithExecOpts{UseEntrypoint: true})
```

Reply:
Moved!